boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/service/EventService.kt

```diff
@@ -188,12 +189,12 @@ class EventService @Autowired constructor(
             }
             if (category != null) {
                 return when (category) {
-                    EventCategory.MUSIC -> "music"
-                    EventCategory.ART -> "miscArt"
-                    EventCategory.TECH -> "tech"
-                    EventCategory.SPORT -> "sport"
-                    EventCategory.ALL -> "???"
-                    EventCategory.OTHER -> null
+                    EventCategory.MUSIC -> "Musik"
```

Comment:
those values are used for the category fallback icons, those are not normal text. so by changing them you broke all the icons :)

Reply:
It has been fixed.